src/movement/test/test_thruster_sabertooth.cpp

```diff
@@ -0,0 +1,120 @@
+#include <gtest/gtest.h>
+#include "ros/ros.h"
+#include "robosub/thruster.h"
+#include "utility/serial_testbench.hpp"
+#include "utility/serial.hpp"
+
+typedef struct thruster_info
+{
+  std::string name;
+  uint8_t address;
+  uint8_t port;      // 0 or 1
+}Thruster_info;
+
+Thruster_info *mThruster_info;
+Serial mSerial;
+
+TEST(SerialSubscriber, basicTest)
```

Comment:
A comment for each test specifying exactly what it does and what it tests would be very helpful.